traffic_ops/traffic_ops_golang/monitoring.go

```diff
@@ -1,90 +1,45 @@
-// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This isn't a monitoring change, this is the `/monitoring` endpoint in Traffic Ops.